advanced_source/super_resolution_with_onnxruntime.py

```diff
@@ -1,10 +1,17 @@
 """
 (optional) Exporting a Model from PyTorch to ONNX and Running it using ONNX Runtime
-========================================================================
+===================================================================================
+
+.. note::
+    As of PyTorch 2.1, there are two versions of ONNX Exporter.
+
+    * ``torch.onnx.dynamo_export`is the newest (still in beta) exporter based on the TorchDynamo technology released with PyTorch 2.0
+    * ``torch.onnx.export`` is based on TorchScript backend and has been available since PyTorch 1.2.0
```

Comment:
```suggestion
    * ``torch.onnx.dynamo_export`is the newest (still in beta) exporter based on the TorchDynamo technology released with PyTorch 2.0.
    * ``torch.onnx.export`` is based on TorchScript backend and has been available since PyTorch 1.2.0.
```